reconcile/utils/terrascript_aws_client.py

```diff
@@ -4514,32 +4513,37 @@ def _get_commit_sha(self, repo_info: Mapping) -> str:
         return ""
 
     def _get_asg_image_id(
-        self, image: Mapping, account: str, region: str
-    ) -> Tuple[Optional[str], str]:
+        self, filters: Iterable[dict[str, Any]], account: str, region: str
```

Comment:
I miss this in my PRs as well, but our guidelines state to use abstract types in favor of concrete ones: https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#abstract-collection-types